Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -893,12 +904,67 @@ struct AttributeData {
                         break;
                     case VertexAttribute.BlendIndices:
                         attributes.JOINTS_0 = accessorId;
+                        accessor.componentType = GLTFComponentType.UnsignedShort;
                         break;
                     default:
                         throw new ArgumentOutOfRangeException();
                 }
             }
 
+            // Add skin
+            if (uMesh.bindposes != null && uMesh.bindposes.Length > 0)
+            {
+                Skin skin = m_SkinMap[uMesh];
+                Transform[] uBones = m_uBones[m_Skins.IndexOf(skin)];
+            
+                var accessor = new Accessor {
+                    byteOffset = 0,
+                    componentType = GLTFComponentType.Float,
+                    count = uBones.Length,
+                    typeEnum = GLTFAccessorAttributeType.MAT4
+                };
+                
+                var accessorId = AddAccessor(accessor);
+                skin.inverseBindMatrices = accessorId;
+                
+                List<uint> joints = new List<uint>();
+                for (int i = 0; i < uBones.Length; i++)
+                {
+                    uint boneIdx = (uint) m_Nodes.FindIndex(node => node.name == uBones[i].name);

```

Comment:
Node lookup cannot rely on GameObject names. Those names are not unique or could be empty strings.
A solution not pretty but still the best way:

- either `GameObjectExport` converts the bones list to a nodeID list itself (if the flow of computation allows for that)...
- or
  - `GameObjectExport` keeps an Transform to node ID map
  -  `GameObjectExport` provides a method to convert a `Transform` to a node ID (looking up that id map)

Rubbing this out also removes the Linq dependency

Reply:
How about doing it in GltfWriter, and changing the signature of "AddNode" to take a Transform, instead of translation, rotation, and scale?

Reply:
`GltfWriter` *should* be agnostic of UnityEngine stuff like `GameObject` or `Transform`. The reason is that one could theoretically feed it with completely different source data (e.g. DOTS Entities) at some point.
Granted, the separation is currently not clean, but it shouldn't get abolished completely.

Reply:
Alright, I'll do another commit to conform

Reply:
Please check if current solution is satisfactory